packages/@aws-cdk/aws-apigateway/test/authorizers/integ.request-authorizer.lit.ts

```diff
@@ -24,6 +24,7 @@ const restapi = new RestApi(stack, 'MyRestApi', { cloudWatchRole: true });
 const authorizer = new RequestAuthorizer(stack, 'MyAuthorizer', {
   handler: authorizerFn,
   identitySources: [IdentitySource.header('Authorization'), IdentitySource.queryString('allow')],
+  resultsCacheTtl: Duration.minutes(5),
```

Comment:
why? This doesn't add anything to this integration test.

Reply:
I edited this test to re-run this integration test and verify that `addToLogicalId()` worked correctly, however I will instead create a separate test

Reply:
I think this is a breaking change so I also tried to add a feature flag